.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 3
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 4
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
```

Comment:
can we add WIP here to keep our work in progress PRs open?

Reply:
The only way of specifying exceptions is by adding labels. There are no rules available around PR title.